.lintrunner.toml

```diff
@@ -57,7 +57,7 @@ init_command = [
   'run',
   'pip_init',
   '--dry-run={{DRYRUN}}',
-  '--requirement=requirements-lintrunner.txt'
+  '--requirement=requirements-dev.txt'
 ]
 is_formatter = true
```

Comment:
pylint is currently disabled because the glob pattern is empty:

```
include_patterns = [
]
```

be sure to include files if you would like to enable the linter.

Reply:
enabled